extras/integration_tests/tests/lib/database/database.dart

```diff
@@ -69,25 +69,25 @@ class PreferenceConverter extends TypeConverter<Preferences, String> {
   }
 }
 
-@UseMoor(
+@DriftDatabase(
   tables: [Users, Friendships],
   queries: {
     'mostPopularUsers': 'SELECT * FROM users u '
-        'ORDER BY (SELECT COUNT(*) FROM friendships '
+        'ORDER BY (SELECT COUNT(*) count FROM friendships '
         'WHERE first_user = u.id OR second_user = u.id) DESC LIMIT :amount',
-    'amountOfGoodFriends':
-        'SELECT COUNT(*) FROM friendships f WHERE f.really_good_friends AND '
-            '(f.first_user = :user OR f.second_user = :user)',
+    'amountOfGoodFriends': 'SELECT COUNT(*) count FROM friendships f '
+        'WHERE f.really_good_friends = 1 AND (f.first_user = :user '
+        'OR f.second_user = :user)',
     'friendshipsOf': ''' SELECT 
           f.really_good_friends, user.**
        FROM friendships f
-         INNER JOIN users user ON user.id IN (f.first_user, f.second_user) AND
-             user.id != :user
+         INNER JOIN users "user" ON "user".id IN (f.first_user, f.second_user) 
+         AND "user".id != :user
        WHERE (f.first_user = :user OR f.second_user = :user)''',
-    'userCount': 'SELECT COUNT(id) FROM users',
+    'userCount': 'SELECT COUNT(id) count FROM users',
```

Comment:
For these (and similar) changes, I think you can also enable the `new_sql_code_generation` [builder option](https://drift.simonbinder.eu/docs/advanced-features/builder_options/#recommended-options) in the `integration_tests/tests` package. It will make the generator add the explicit column name for you and we don't need to manually change the SQL here.

Reply:
Okay, I reverted the database file and enable new sql generator, but I had to change it: `user` is reserved keyword in Postgres, so it is necessary to enclose it in quotation marks. Second problem is, boolean treated as integer right now, and postgres can't accept use as boolean without equal operator
```sql
SELECT COUNT(*) FROM friendships f
WHERE f.really_good_friends
AND (f.first_user = :user OR f.second_user = :user)
```
This sql works with SQLite because it can threat integer as boolean (`f.really_good_friends` can be used without ` = 1`). But Postgres can't handle that, so it throws `argument of AND must be type boolean, not type integer` error. 
Custom query parser don't know that `really_good_friends` is boolean or not. 

1. solution is leave it as is and users have to update existing SQL queries (breaks backward compatible)
2. solution is modify table generation and query builder code to handle booleans differently. Latter is much more work